implants/lib/eldritch/Cargo.toml

```diff
@@ -70,6 +69,9 @@ winreg = { workspace = true }
 [target.'cfg(not(windows))'.dependencies]
 pnet = { workspace = true }
 
+[target.'cfg(not(target_os = "freebsd"))'.dependencies]
```

Comment:
Do we want to support all the BSDs?
- "freebsd"
- "openbsd"
- "netbsd"

Reply:
I can check if my changes work on the other BSDs, but not sure if it will

Reply:
Yeah lmk how it goes.
If it doesn't we should add them to the exclude otherwise what a W!

Reply:
We will only be supporting freebsd at this time.